test/bcountermgr_SUITE.erl

```diff
@@ -0,0 +1,134 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2014 SyncFree Consortium.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+-module(bcountermgr_SUITE).
+
+-compile({parse_transform, lager_transform}).
+
+%% common_test callbacks
+-export([init_per_suite/1,
+         end_per_suite/1,
+         init_per_testcase/2,
+         end_per_testcase/2,
+         all/0]).
+
+%% tests
+-export([new_bcounter_test/1,
+         test_dec_success/1,
+         test_dec_fail/1,
+         test_dec_multi_success/1]).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("kernel/include/inet.hrl").
+
+-define(TYPE, antidote_crdt_bcounter).
+
+
+init_per_suite(Config) ->
+    test_utils:at_init_testsuite(),
+    Clusters = test_utils:set_up_clusters_common(Config),
+    Nodes = lists:flatten(Clusters),
+
+    %Ensure that the clocksi protocol is used
```

Comment:
Don't you need the local validation from clock si?  Otherwise two local transactions could overuse the permissions?

I guess it doesn't have to be clock_si specifically, but it needs si though?